docs/reference/behavioral-analytics/apis/delete-analytics-collection.asciidoc

```diff
@@ -59,3 +51,4 @@ The following example deletes the Analytics Collection named `my_analytics_colle
 ----
 DELETE _application/analytics/my_analytics_collection/
 ----
+// TEST[skip:Behavioral Analytics APIs emit deprecation warnings and will not be updated]
```

Comment:
There appears to be a bug in our test framework as emitting warnings during test setup can't be handled. Since we're not going to be updating these APIs my solution was to skip the tests.